R_vs_Python_comparison.py

```diff
@@ -0,0 +1,117 @@
+import argparse
```

Comment:
Add imports from the standard Python library first, then empty line then other, from pip-installed libraries